components/server/package.json

```diff
@@ -18,9 +18,9 @@
     "clean:node": "rimraf node_modules",
     "purge": "yarn clean && yarn clean:node && yarn run rimraf yarn.lock",
     "lint": "yarn tslint -p .",
-    "test": "mocha --opts mocha.opts './**/*.spec.ts' --exclude './node_modules/**'",
-    "db-test": "r(){ . $(leeway run components/gitpod-db:db-test-env); yarn db-test-run; };r",
-    "db-test-run": "mocha --opts mocha.opts '**/*.spec.db.ts' --exclude './node_modules/**'",
+    "test": "yarn db-test",
+    "db-test": "r(){ . $(leeway run components/gitpod-db:db-test-env); leeway build components/gitpod-db/go:init-testdb; yarn db-test-run; };r",
```

Comment:
That target does not work.

AFAIK we use `leeway run components/gitpod-db:init-testdb` for interactive cases, no? :thinking:

Reply:
What fails when you run `yarn test`? For me, this executes fine in non-interactive mode

Reply:
the `HOST` confusion we have ongoing for months already :upside_down_face: Points to localhost instead of `127.0.0.1`